src/sys/windows/selector.rs

```diff
@@ -357,10 +357,11 @@ impl Selector {
 }
 
 cfg_net! {
-    use super::SocketState;
+    use super::{SocketState, InternalState};
     use crate::Token;
     use std::os::windows::io::AsRawSocket;
 
+
```

Comment:
Extra line!

Reply:
This is the downside of having a large amount of code inside a macro calls; no rustfmt :(

Reply:
Yea pretty annoying 😔

Reply:
Yea pretty annoying 😔